tests/oss-fuzz/avif_decode_fuzzer.cc

```diff
@@ -32,7 +32,8 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t * Data, size_t Size)
     if (result == AVIF_RESULT_OK) {
         for (int loop = 0; loop < 2; ++loop) {
             while (avifDecoderNextImage(decoder) == AVIF_RESULT_OK) {
-                if ((loop != 0) || (decoder->imageIndex != 0)) {
+                if (((decoder->image->width * decoder->image->height) > (47 * 1024 * 1024)) || (loop != 0) ||
+                    (decoder->imageIndex != 0)) {
                     // Skip the YUV<->RGB conversion tests, which are time-consuming for large
                     // images. It suffices to run these tests only for loop == 0 and only for the
                     // first image of an image sequence.
```

Comment:
Adapt comment, such as

> only for the first image of an image sequence of reasonable dimensions.

Reply:
Thank you for the suggestion. I did consider whether this comment should be updated, and thought that the first sentence "Skip the YUV<->RGB conversion tests, which are time-consuming for large images." could be interpreted as describing the new image size check. So I didn't modify the comment.

Reply:
Yes, I agree it is better to fuzz the RGB-YUV conversion functions directly. Note that we will still need to skip that test for large images when the conversion function uses libavif's built-in implementation, because each fuzzer test needs to run for 100 times in no more than 90 seconds. (Google's internal fuzzing framework runs each fuzzer test 100 times with a 90 seconds timeout.)